src/app/components/select-field/index.html

```diff
@@ -166,31 +166,32 @@
       propertyName="data"
       isOptional="true"
     >
-      An array of <a stacheRouterLink="." fragment="skyselectfield-properties">The <sky-code>SkySelectField</sky-code> objects</a> provided to the select field's <sky-code>data</sky-code> property. Use this property to populate your custom picker with options for the user to select.
+      Receives an array <a stacheRouterLink="." fragment="skyselectfield-properties">of <sky-code>SkySelectField</sky-code> objects</a> from the select field's <sky-code>data</sky-code> property. Use this property to populate the custom picker with items for users to select.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="headingText"
       isOptional="true"
     >
-      The text provided to the select field's <sky-code>headingText</sky-code> property. If your custom picker is a modal, use this property for the modal's title.
+      Receives text from the select field's <sky-code>pickerHeading</sky-code> property. Use this property to display a header in the custom picker's modal.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="selectedValue"
       isOptional="true"
     >
-      The select field's current value. If the select field's <sky-code>selectMode</sky-code> is set to <sky-code>single</sky-code>, this will be a <sky-code>SkySelectField</sky-code> object; otherwise it will be an array of <sky-code>SkySelectField</sky-code> objects. Use this property to select the corresponding items in the custom picker.
+      Receives the select field's current value. If the select field's <sky-code>selectMode</sky-code> is set to <sky-code>single</sky-code>, then this value is <a stacheRouterLink="." fragment="skyselectfield-properties">a <sky-code>SkySelectField</sky-code> object</a>. If the select field's <sky-code>selectMode</sky-code> is set to <sky-code>multiple</sky-code>, then this value is an array <a stacheRouterLink="." fragment="skyselectfield-properties">of <sky-code>SkySelectField</sky-code> objects</a>. Use this property to select the corresponding items in the custom picker.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="selectMode"
       isOptional="true"
     >
-      The value provided to the select field's <sky-code>selectMode</sky-code> property. Use this property to determine whether the custom picker should allow more than one value to be selected.
+      Receives a <sky-code>SkySelectFieldSelectMode</sky-code> value from the select field's <sky-code>selectMode</sky-code> property. Use this property to specify the selection mode that determines whether users can select one item or multiple items.
```

Comment:
The change from "use this property to determine whether" to "use this property to specify" alters the meaning of the property. The consumer should be looking at the property's value to determine the action to take, but the edit makes it sound like the consumer should be specifying the value. How about this?

"The <sky-code>SkySelectFieldSelectMode</sky-code> value from the select field's <sky-code>selectMode</sky-code> property. Use this property to determine whether users can select one item or multiple items."